src/Core/Migration/V6_7/Migration1737430168RemoveFileTypeOfDocumentTable.php

```diff
@@ -19,6 +19,7 @@ public function getCreationTimestamp(): int
 
     public function update(Connection $connection): void
     {
+        $connection->executeStatement('ALTER TABLE `document` CHANGE COLUMN `file_type` `file_type` VARCHAR(255) NULL');
```

Comment:
Are we allowed to change existing migrations that were already included in a release (in this case an RC release)? Because Imagine someone is on 6.7 RC5 and updates to 6.7 final, this will not execute right? So he will still have the bug.

Might be better to add a new migration for this 🤔

Reply:
updating from RC to final is not supported, afaik